python/Fair_Rations.py

```diff
@@ -0,0 +1,119 @@
+#!/bin/python3
+
+import math
+import os
+import random
+import re
+import sys
+
+# Complete the fairRations function below.
+def fairRations(B):
+    suml = 0;
+    current = -1
+    paired = -1
+    distance = 0
+    total = 0;
+
+    for i in B:
+        suml += i
+        if(i % 2 == 1 and current == -1):
+            current = i
+            paired = -1
+            distance = 0
+        elif(current != -1 and i % 2 == 1):
+            paired = i
```

Comment:
Codacy found an issue: [local variable 'paired' is assigned to but never used (F841)](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6294551)